tools/AzureRM/AzureRM.psd1

```diff
@@ -87,6 +87,7 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '4.0.0';
                @{ModuleName = 'AzureRM.Network'; RequiredVersion = '5.0.0'; }, 
                @{ModuleName = 'AzureRM.NotificationHubs'; RequiredVersion = '4.0.0'; }, 
                @{ModuleName = 'AzureRM.OperationalInsights'; RequiredVersion = '4.0.0'; }, 
+               @{ModuleName = 'AzureRM.PowerBI'; RequiredVersion = '1.0.0'; }, 
```

Comment:
@selasagi same comment - I thought the cmdlets would be shipped with `AzureRM.PowerBIEmbedded`?

Reply:
correct